src/user/message.ts

```diff
@@ -566,6 +567,32 @@ export class Message extends Accessory implements Sayable {
     }
   }
 
+  /**
+   * Recall a message.
+   * > Tips:
+   * @returns {Promise<boolean>}
+   *
+   * @example
+   * const bot = new Wechaty()
+   * bot
+   * .on('message', async m => {
+   *   const recallMessage = await msg.say('123')
+   *   const isSuccess = await recallMessage.recall()
```

Comment:
recall should need a messageId.

Reply:
I don't think so, this is a function on `Message` class, it is recalling the message which called this function.

Reply:
I don't think so, this is a function on `Message` class, it is recalling the message which called this function.

Reply:
ok then, I send two messages. what should I do for I want to recall the first one?
```
const recallMessage = await msg.say('123')
const recallMessage2 = await msg.say('456')
const isSuccess = await recallMessage.recall()
```
like this?

Reply:
yes.